### 1. 
Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].
Найдите ее среднее значение и дисперсию.

In [1]:
a= 200
b=800
M = (a+b)/2
D = ((b-a)**2)/12
M, D

(500.0, 30000.0)

In [2]:
from scipy import stats

In [3]:
stats.uniform.mean(a, 600)

500.0

In [4]:
stats.uniform.var(a, 600) 

30000.0

### 2. 
О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.
Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?
Если да, найдите ее.


Да
* 0.2 = ((b-0.5)** 2)/12
* 2.4 = (b-0.5)** 2
* 1.5491 = b-0.5
* b=2,049

In [5]:
stats.uniform.var(0.5, 1.5491) 

0.1999759008333333

### 3. 
Непрерывная случайная величина X распределена нормально и задана плотностью распределения
f(x) = (1 / (4 * sqrt(2*pi))) * (exp(-(x+2)**2) / 32).
Найдите:
а). M(X)
б). D(X)
в). std(X) (среднее квадратичное отклонение)


а). M(X) = -2

б). D(X) = 16

в). std(X) (среднее квадратичное отклонение) = 4 


### 4. 
Рост взрослого населения города X имеет нормальное распределение.
Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см.
Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
а). больше 182 см
б). больше 190 см
в). от 166 см до 190 см
г). от 166 см до 182 см
д). от 158 см до 190 см
е). не выше 150 см или не ниже 190 см
ё). не выше 150 см или не ниже 198 см
ж). ниже 166 см.


In [6]:
from IPython.display import Image
from IPython.core.display import HTML 
Image(url= 'https://www.center-pss.ru/img/normraspr.jpg')

In [7]:
#больше 182 см

Z = (182 - 174)/8
Z

1.0

In [8]:
1 - 0.8413

# Перепроерка:  промежутке от -1 сигму до +1 сигмы лежит 68% наблюдений
# 182 - расстояние одной сигмы. Т.е (100 - 68) / 2, тк графики семитричны. Отет 16

0.15869999999999995

б). больше 190 см = (190 - 174) / 8 = 2; 1 - 0.9772 = **0.0228** Тоже что (100 - 95.4) / 2 = 2.3% 

в). от 166 см до 190 см = (166 - 174) / 8 = -1; 1 - 0.8413 = 0.1586; 0.9772 - 0.1586 = **0.8186**

г). от 166 см до 182 см = 0.8413 - 0.1587 = **0.6825**

д). от 158 см до 190 см  = (158 - 174) / 8 = -2; (190 - 174) / 8 = 2; 0.9772 - 0.0228 = *0.9492**

е). не выше 150 см или не ниже 190 см = (150 - 174) / 8 = -3; 0.0014 + 0.0228 = **0.0242**

ё). не выше 150 см или не ниже 198 см =0.0014 * 2 = **0.0028**

ж). ниже 166 см. = **0.1587**

### 5. 
На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от
математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?


In [9]:
(190-178)/5

2.4